### ISSUE TITLE
About BSDS Dataset

### ISSUE THREAD

**Issue by zhoujh2020** [ZhouCX117/UAED#2]
We noticed that you increased the BSDS dataset by 100 times, where the increase method included rotating and cropping the largest rectangle. Our question is that the pictures in the training set will no longer have the same size, and it will not be possible to train with a batch size of 4. Did you do an additional random cropping operation on the dataset to get the same size?

**Comment by ZhouCX117** [comment]
Hi, thanks for your interest. We use the processed data from the 2018 ECCV paper "Learning to Predict Crisp Boundaries".  We will ask permission from Ruoxi Deng and then release the data.

**Comment by zhoujh2020** [comment]
Thank you very much for your prompt reply. Can you give us some details first? We noticed that you are using 25 angles in your paper, while only 16 angles are used in LPCB. Have you made any changes based on LPCB?  Also, random scaling is done in LPCB, did you do that too?

**Comment by ZhouCX117** [comment]
I have uploaded the used code and processed dataset, and the data loader file is also updated.

**Comment by zhoujh2020** [comment]
Thank you very much for providing these data so quickly.

**Comment by zhoujh2020** [comment]
Sorry to bother you again, but it seems you have a problem with your train_val_all.lst. There is some pictures in the test set in this list, it may be that there is a mistake when you generated the .lst file. Also, you didn't provide the val-augmented dataset. Thank you for your help.

**Comment by ZhouCX117** [comment]
What does "some pictures in the test set in this list" mean? There are 200 pictures in the train file and 100 pictures in the val file. After processing the data, we get 30000 samples. The val-augmented dataset is obtained by the provided Matlab code(1.images/rotate_imgs.m; 2.run_gen_dataset.m). It may cost some time for me to provide the whole dataset and code.

**Comment by JaniniRami** [comment]
> Sorry to bother you again, but it seems you have a problem with your train_val_all.lst. There is some pictures in the test set in this list, it may be that there is a mistake when you generated the .lst file. Also, you didn't provide the val-augmented dataset. Thank you for your help.

Hey I am facing the same issue, where some files in the train_val_all.lst file does not exist in the dataset that is downloaded from the baidu disk link provided, did you find a way to fix it?

**Comment by ZhouCX117** [comment]
I have uploaded the complete BSDS dataset, and you can have a try.

**Comment by ZhouCX117** [comment]
I heve uploaded the complete dataset, and you can have a try.
















At 2023-06-28 04:30:23, "Rami Janini" ***@***.***> wrote:

Sorry to bother you again, but it seems you have a problem with your train_val_all.lst. There is some pictures in the test set in this list, it may be that there is a mistake when you generated the .lst file. Also, you didn't provide the val-augmented dataset. Thank you for your help.

Hey I am facing the same issue, where some files in the train_val_all.lst file does not exist in the dataset that is downloaded from the baidu disk link provided, did you find a way to fix it?
